src/main/java/uk/gov/hmcts/juror/api/moj/service/audit/JurorAuditServiceImpl.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.juror.api.moj.service.audit;
+
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.PersistenceContext;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.juror.api.moj.audit.dto.JurorAudit;
+import uk.gov.hmcts.juror.api.moj.audit.dto.QJurorAudit;
+import uk.gov.hmcts.juror.api.moj.service.JurorPoolService;
+import uk.gov.hmcts.juror.api.moj.utils.DateUtils;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class JurorAuditServiceImpl implements JurorAuditService {
+
+    @PersistenceContext
+    EntityManager entityManager;
+
+    private final JurorPoolService jurorPoolService;
+
+
+    JPAQueryFactory getQueryFactory() {
+        return new JPAQueryFactory(entityManager);
+    }
+
+    @Override
+    public List<JurorAudit> getAllAuditsFor(List<String> jurorNumbers) {
+        return getQueryFactory()
+            .selectFrom(QJurorAudit.jurorAudit)
+            .where(QJurorAudit.jurorAudit.jurorNumber.in(jurorNumbers))
+            .orderBy(QJurorAudit.jurorAudit.revisionInfo.timestamp.asc())
+            .fetch();
+    }
+
+    @Override
+    public List<JurorAudit> getAllAuditsChangedBetweenAndHasCourt(LocalDate fromDate, LocalDate toDate,
+                                                                  List<String> locCodes) {
+        List<JurorAudit> data = getQueryFactory()
```

Comment:
could we not do this in one query by joining to jurorpool, pool tables?

Reply:
Agreed, this was left over from when we were using AuditQueries instead of QueryDSL for pulling back the audit information.

I have updated :)